Dining/templatetags/dining_tags.py

```diff
@@ -1,5 +1,26 @@
 from django import template
 
+from Dining.models import DiningEntry
+
 register = template.Library()
 
-# Todo: this file is currently not used and can maybe be deleted
+
+@register.filter(name='can_join_slot')
```

Comment:
```suggestion
@register.filter
```

`name` argument is not needed, defaults to name of function